src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -82,6 +82,13 @@ private function configureRoutes(RoutingConfigurator $routes): void
         } else {
             $routes->import($configDir.'/{routes}.php');
         }
+
+        if (
+            false !== ($fileName = (new \ReflectionClass(static::class))->getFileName()) &&
```

Comment:
we usually put the `&&` on the next line

Reply:
in other lines on this file, we just use `(new \ReflectionObject($this))->getFileName()` without the extra checks btw

Reply:
What if it's an anonymous class or something weird like that?

Reply:
it's still going to be declared in a file so this should work